dtfs-central-api/src/constants/activityConstants.js

```diff
@@ -0,0 +1,14 @@
+const PORTAL_ACTIVITY_LABEL = {
```

Comment:
can we change this to `exports.PORTAL_ACTIVITY_LABEL = {...}`
we don't need to define it as a variable at the top.

Reply:
done